pkg/broker/broker.go

```diff
@@ -959,3 +965,11 @@ func ocLogin(log *logging.Logger, args ...string) error {
 	}
 	return nil
 }
+
+func extractNameFromPlanID(planID string) (string, error) {
+	splitPlan := strings.Split(planID, "-")
```

Comment:
Can you get a planID without a '-' in it?

Reply:
@rthallisey not the way I generated them. There would always be at least one '-' in it.